core/ast_node.js

```diff
@@ -250,7 +250,7 @@ Blockly.ASTNode.prototype.findNextEditableField_ = function(location,
   var fieldRow = parentInput.fieldRow;
   var fieldIdx = fieldRow.indexOf(location);
   var startIdx = opt_first ? 0 : fieldIdx + 1;
-  for (var i = startIdx; i < fieldRow.length; i++) {
+  for (var i = startIdx, field; field = fieldRow[i]; i++) {
     var field = fieldRow[i];
```

Comment:
This line is redundant.  :)

Reply:
Shoot missed that. Will fix in my next PR.